src/@next/Icon/icons/icons.ts

```diff
@@ -164,6 +164,7 @@ export const iconNames = [
   'ri-twitter-line',
   'ri-upload-fill',
   'ri-upload-line',
+  'ri-user-add-fill',
   'ri-user-line',
```

Comment:
hi @ninariccimarie what about user-add-line?

Reply:
@chiahou do we need it? we usually just add what we need

Reply:
as discussed, we need the user-add-line 👍 I've included it in @chiahou